cmd/health-agent/probers.go

```diff
@@ -15,6 +16,7 @@ import (
 
 func setupProbers() (*proberlist.ProberList, error) {
 	pl := proberlist.New("/probers")
+	go pl.CreateAndAdd(aws.Register, "/sys/aws", 0)
```

Comment:
On a machine which is unable to contact the metadata service (perhaps because it's not an AWS instance), this will result in an empty /sys/aws directory. Instead, no directory should be created.